msticpy/data/drivers/local_velociraptor_driver.py

```diff
@@ -0,0 +1,222 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""Local Velociraptor Data Driver class."""
+import logging
+from collections import defaultdict
+from functools import lru_cache
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+import pandas as pd
+from tqdm.auto import tqdm
+
+from ..._version import VERSION
+from ...common.exceptions import MsticpyDataQueryError
+from ...common.provider_settings import get_provider_settings
+from ...common.utility import export, valid_pyname
+from .driver_base import DriverBase, QuerySource
+
+__version__ = VERSION
+__author__ = "ianhelle"
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=too-many-instance-attributes
+@export
+class VelociraptorLogDriver(DriverBase):
+    """OSQueryLogDriver class to execute kql queries."""
+
+    OS_QUERY_DATEIME_COLS = {
```

Comment:
Given the name are these still valid for Velociraptor?

Reply:
both comment and OS_QUERY_DATEIME_COLS variable don't apply to velociraptor so likely not.